iml-system-tests/tests/common/vagrant.rs

```diff
@@ -190,6 +162,174 @@ pub async fn setup_deploy_servers(
     Ok(())
 }
 
+pub async fn setup_deploy_docker_servers(
+    config: &ClusterConfig,
+    profile: &str,
+) -> Result<(), Box<dyn std::error::Error>> {
+    setup_bare(&[&config.storage_servers()[..], &[config.iscsi]].concat()).await?;
+
+    iml::server_add(&config.storage_servers()[..], profile)
+        .await?
+        .checked_status()
+        .await?;
+
+    halt().await?.args(config.all()).checked_status().await?;
+
+    for host in config.all() {
+        snapshot_save(host, "servers-deployed")
+            .await?
+            .checked_status()
+            .await?;
+    }
+
+    up().await?.args(config.all()).checked_status().await?;
+
+    Ok(())
+}
+
+async fn check_ntp_status(host: &str) -> Result<bool, io::Error> {
+    let x = vagrant()
+        .await?
+        .arg("ssh")
+        .arg("-c")
+        .arg("'sudo ntpstat'")
+        .arg(host)
+        .status()
+        .await?;
+
+    Ok(x.success())
+}
+
+async fn restart_ntpd(hosts: &[&str]) -> Result<(), io::Error> {
+    vagrant()
+        .await?
+        .arg("ssh")
+        .arg("-c")
+        .arg("'sudo systemctl restart ntpd.service'")
```

Comment:
Are single quotes really needed?

Reply:
I can try removing them. Not sure to be honest. In general, they are used after `-c`

Reply:
I thought the quotes were only needed when constructing a shell command. But this looks more like `execve()`.

Reply:
I'm going to remove them and run a test to see if it works.

Reply:
I added this to the Vagrantfile so we can configure ntp using a provisioning script so it's no longer an issue :)